bundle-workflow/python/sign.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+import subprocess as sp
+import tempfile
+import uuid
+import argparse
+from manifests.bundle_manifest import BundleManifest
+
+parser = argparse.ArgumentParser(description = "Sign artifacts")
+parser.add_argument('manifest', type = argparse.FileType('r'), help="Manifest file.")
+args = parser.parse_args()
+
+def sign(filename):
+    signatureFile = filename + ".asc"
+    signing_cmd = './opensearch-signer-client -i ' + filename + ' -o ' + signatureFile + ' -p pgp'
+    sp.run(signing_cmd.split(),cwd='opensearch-signer-client/src')
+    verify_cmd = 'gpg --verify-files ' + signatureFile
+    sp.run(verify_cmd.split(),cwd='opensearch-signer-client/src')
+    sp.run(['mv',signatureFile,'signed_artifacts/'])
+
+manifest = BundleManifest.from_file(args.manifest)
+output_dir = os.path.join(os.getcwd(), 'signed_artifacts')
+os.makedirs(output_dir, exist_ok = True)
+build_id = os.getenv('OPENSEARCH_BUILD_ID', uuid.uuid4().hex)
+
+os.system('git clone https://github.com/opensearch-project/opensearch-signer-client.git')
```

Comment:
Wrap this underneath signer, and use existing Git classes to avoid having to do explicit `git clone` or `rm -rf` signer at the end.